modules/database/src/database.proto

```diff
@@ -110,12 +110,19 @@ message DropCollectionRequest {
     bool deleteData = 2;
 }
 
+message MigrateRequest {
+    string schemaName = 1;
+}
+
+message MigrateResponse {}
```

Comment:
rename to Empty. This is what we use in all other protofiles that provide empty responses